pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java

```diff
@@ -27,10 +25,21 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.annotation.Nonnull;
-import java.util.Map;
-
-import static java.util.stream.Collectors.toMap;
+import com.google.common.base.Charsets;
+import com.google.common.hash.HashCode;
+import com.google.common.hash.Hashing;
+import com.ibm.icu.text.DateFormat;
+import com.redhat.pantheon.asciidoctor.extension.HtmlModulePostprocessor;
+import com.redhat.pantheon.asciidoctor.extension.MetadataExtractorTreeProcessor;
+import com.redhat.pantheon.asciidoctor.extension.SlingResourceIncludeProcessor;
+import com.redhat.pantheon.conf.GlobalConfig;
+import com.redhat.pantheon.extension.HydraIntegration;
+import com.redhat.pantheon.model.ProductVersion;
+import com.redhat.pantheon.model.module.Content;
+import com.redhat.pantheon.model.module.Metadata;
+import com.redhat.pantheon.model.module.Module;
+import com.redhat.pantheon.model.module.ModuleVersion;
+import com.redhat.pantheon.sling.ServiceResourceResolverProvider;
```

Comment:
massive import order change :disappointed: 
We should align all of our editors... somehow, as these changes obfuscate things

Reply:
yes! I need start using IntelliJ.